community/contributors.md

```diff
@@ -75,5 +75,6 @@ us know in chat or team meeting.
 
 * **Neuroglia**
     * Charles d'Avernas
+    * Jean-Baptiste Bianchi
```

Comment:
@JBBianchi could you check wether this is correct ?

Reply:
It is, thanks!